packages/local-find/src/UploadModal/styles.js

```diff
@@ -141,4 +141,13 @@ export default (theme) => ({
   helpIconButton: {
     marginBottom: 10,
   },
+  submitBtn: {
```

Comment:
@huuaho Can we pass this color as Prop instead of hard coding it in package? If no value provided for these props bento colors are default value.

Reply:
That is the way it is currently set up. These are the default colors and can be overriden. 

Here is an example of the style being overriden with the accompanying PR on the frontend 

https://github.com/CBIIT/bento-cds-frontend/blob/ff6552c1d8c1d9d9c67156356523539aa141777d/src/pages/dashTemplate/sideBar/BentoFacetFilterStyle.js#L488

Reply:
Got it thanks @huuaho.